barcodes_gs1_label/models/option_config.py

```diff
@@ -0,0 +1,156 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import re
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class BarcodesGS1LabelOptionsConfig(models.Model):
+    _name = "barcodes.gs1.label.options.config"
+
+    name = fields.Char(string="Label Config", required=True, translate=True)
+
```

Comment:
```suggestion
company_id = fields.Many2one(comodel_name="res.company", required=True, default=seld.env.user.company_id)
```
Add the company, this configurations should be company aware, show it in the view and add the company permission rule (check the existing rules as an example)